dbfit-java/build.gradle

```diff
@@ -222,3 +251,114 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact.
```

Comment:
> now we rely it's at the end of the list

We could simply remove every artifact:

`configurations.archives.artifacts.with { archives ->
    archives.each {
        remove(it)
    }
}`

Do we need to even check the file type/name at all?

If we don't mind relying on the configuration only ever having one default artefact then yes let's condense to a *one-liner*.

In your snippet, how does the `reverse.find` work? What does it do?